sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -54,6 +63,13 @@ public IEnumerable<TestSuite> BuildFrom(ITypeInfo typeInfo)
 
         public IEnumerable<TestSuite> BuildFrom(ITypeInfo typeInfo, IPreFilter filter)
         {
+            var latestVersion = _serviceVersions.Any() ? _serviceVersions.Max(s => Convert.ToInt32(s)) : (int?)null;
+            _actualPlaybackServiceVersion = RecordingServiceVersion != null ? Convert.ToInt32(RecordingServiceVersion) : latestVersion;
+            _actualLiveServiceVersion =
+                LiveServiceVersion != null ?
+                    Convert.ToInt32(LiveServiceVersion) :
+                    OnlyTestLatestServiceVersionLazy.Value ? latestVersion : null;
```

Comment:
This has some surprise factor. If both `LiveServiceVersion` and env variable feeding `OnlyTestLatestServiceVersionLazy` is specified then somebody might have hard time figuring out what's going on (depending what they want vs what is set).
At minimum we should throw if both are specified. But if we pursue changing `LiveServiceVersion` concept from previous comment above then this problem might disappear.

Reply:
Changed the `LiveServiceVersions` to an array and now OnlyTestLatestServiceVersionLazy  just picks the latest out of array items.